lib/workers/repository/onboarding/pr/config-description.spec.ts

```diff
@@ -37,8 +36,20 @@ describe('workers/repository/onboarding/pr/config-description', () => {
       config.labels = ['renovate', 'deps'];
       config.schedule = ['before 5am'];
       const res = getConfigDesc(config);
-      // FIXME: explicit assert condition
-      expect(res).toMatchSnapshot();
+      expect(res).toMatchInlineSnapshot(`
```

Comment:
This is also easily overwritten by `jest -u`. 😏

Reply:
Yes, however I'm not sure what part of the string should be explicitly tested. In this instance, I believe that inline snapshots less likely may be overwritten with invalid values.